packages/components/stories/dropdown-list/README.md

```diff
@@ -26,20 +25,19 @@ const DummyComponent = props => (
-| ref               | {object}     | null        | `ref` for the main DropdownList and inner elements. An object of {dropdown_ref, list\_item\_ref, list\_wrapper\_ref} |
-| onScrollStop      | {function}   | null        | Function triggers when the user stops scrolling                          |
-| not\_found\_text  | {string}     | null        | For AutoComplete if the entered text is not found in the list of items  |
-
+| Name            | Type                              | Default   | Description                                                                                                                |
+| --------------- | --------------------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------- |
+| is_visible      | {boolean}                         | null      | Define the visibility of the DropdownList                                                                                  |
+| list_items      | [string] \| [object<text, value>] | null      | List of the items                                                                                                          |
+| onItemSelection | {function}                        | null      | Function returns the selected item                                                                                         |
+| list_height     | {string}                          | `220px`   | Height of the list                                                                                                         |
+| active_index    | {number}                          | null      | Index of the active index                                                                                                  |
+| setActiveIndex  | {function}                        | null      | Function returns the active index when it changes                                                                          |
+| style           | {object}                          | null      | Style object for the main wrapper                                                                                          |
+| dropdown_refs   | {object}                          | null      | `dropdown_refs` for the main DropdownList and inner elements. An object of {dropdown_ref, list_item_ref, list_wrapper_ref} |
+| onScrollStop    | {function}                        | null      | Function triggers when the user stops scrolling                                                                            |
+| not_found_text  | {string}                          | null      | For AutoComplete if the entered text is not found in the list of items                                                     |
+| ref             | {object}                          | undefined | `forwardedRef` for the main `DropdownList`                                                                                 |
```

Comment:
this can be removed because DropdownList doesn't accept or forwards ref any more :)